README.md

```diff
@@ -164,6 +164,46 @@ FAQ
     * **A:** Check your clightning logs with `journalctl -eu clightning`. Do you see something like `bitcoin-cli getblock ... false` failed? Are you using pruned mode? That means that clightning hasn't seen all the blocks it needs to and it can't get that block because your node is pruned. If you're just setting up a new node you can `systemctl stop clightning` and wipe your `/var/lib/clightning` directory. Otherwise you need to reindex the Bitcoin node.
 * **Q:** My disk space is getting low due to nix.
     * **A:** run `nix-collect-garbage -d`
+* **Q:** How do I connect to my nix-bitcoin node through the ssh Tor Hidden Service?
+    * **A:** 
+
+	1. Run `nodeinfo` and note the `SSHD_ONION`
```

Comment:
... on your nix-bitcoin node perhaps?

Reply:
Fixed in be3d4ed

Reply:
Also added a code snippet to make "run nodeinfo" more clear.